include/boost/geometry/strategies/spherical/intersection.hpp

```diff
@@ -145,11 +145,12 @@ struct ecef_segments
     <
         typename UniqueSubRange1,
         typename UniqueSubRange2,
-        typename Policy
+        typename Policy,
+        typename SideStrategy
     >
     static inline typename Policy::return_type
         apply(UniqueSubRange1 const& range_p, UniqueSubRange2 const& range_q,
-              Policy const&)
+              Policy const&, SideStrategy const&)
```

Comment:
So side strategy is not used in all intersection strategies. Only in cartesian one? So side strategy is passed here it is ignored? This is inconsistent. But I don't understand why is this needed at all (see other comment).